### PR TITLE
RHTAPREL-671: Deploy tenants-config on RH stage

### PR DESCRIPTION
**Pull request by gbenhaim**

### CODE REVIEW

File: argo-cd-apps/overlays/staging/kustomization.yaml [hunk 1]
```diff
@@ -4,3 +4,16 @@ resources:
   - ../../base/host
   - ../../base/member
   - ../../base/all-clusters
+  - ../../base/rh-managed-workspaces-config
+  - ../../base/tenants-config
+patches:
+  - path: tenatns-config-managed.yaml
```
Comment (scoheb) [inline argo-cd-apps/overlays/staging/kustomization.yaml:10]: typo in file name

File: argo-cd-apps/overlays/staging/kustomization.yaml [hunk 2]
```diff
@@ -4,3 +4,16 @@ resources:
...
+    target:
+      kind: ApplicationSet
+      version: v1alpha1
+      name: rh-managed-workspaces-config
+  - path: tenatns-config-tenant.yaml
```
Comment (scoheb) [inline argo-cd-apps/overlays/staging/kustomization.yaml:15]: typo in file name

**Review by scoheb — CHANGES_REQUESTED** [review]
file names

File: argo-cd-apps/overlays/staging/kustomization.yaml (hunk 2 above)
Comment (gbenhaim) [reply argo-cd-apps/overlays/staging/kustomization.yaml]: done

File: argo-cd-apps/overlays/staging/kustomization.yaml (hunk 1 above)
Comment (gbenhaim) [reply argo-cd-apps/overlays/staging/kustomization.yaml]: done

**Review by scoheb — APPROVED** [review]

**Comment by gbenhaim** [comment]
@scoheb it needs lgtm

**Review by scoheb — APPROVED** [review]
/lgtm

**Comment by arewm** [comment]
@scoheb @gbenhaim, is this PR still needed?

**Comment by scoheb** [comment]
closing due to inactivity.

**Pull request closed without merging**